lab-judy/server.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const net = require('net');
+const EE = require('events');
+
+
+
+const Client = require('./model/client.js');
+
+const PORT = process.env.PORT || 3000;
+
+const pool = [];
+const nickNamesArray = [];
+const server = net.createServer();
+const ee = new EE();
+
+ee.on('\\user', function(client, string){
+  client.nickname = string.trim();
+  nickNamesArray.push(client.nickname);
+  console.log(nickNamesArray);
+});
+
+
+ee.on('\\all', function(client, string){
+  pool.forEach( c => {
+    c.socket.write(`${client.nickname}: ` + string);
```

Comment:
The message is being broadcasted, but I'm not getting nicknames displaying at all before the message string. I think it's the way you're trying to combine string interpolation and concatenation. This should work:
```
c.socket.write(`${client.nickname}: ${string}`);
```